ios/Podfile.lock

```diff
@@ -566,4 +566,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c34e2287a9ccaa606aeceab922830efb9a6ff69a
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
Keep this as `1.9.3` since it's the latest version

Reply:
How about 1.10.0